File: metrica.c
#include<stdio.h>

main(){
	
	int caso,visitas,i,j,k,cont=0;
	scanf("%d",&caso);

	for(i=1; i<=caso;i++){
		scanf("%d",&visitas);
		
		int Ai[visitas];
	
		for(j=0;j<visitas;j++){
			scanf("%d",&Ai[j]);
			printf("%d",Ai[j]);
			
			for(k=0;k<=j;k++){
				if(k==0){
					cont++;
					printf("%d\n",cont);
					system("pause");
				} else if(Ai[j]!=Ai[k]){
					cont++;
					printf("%d\n",cont);
				}
			}
		}
		printf("Caso %d: %d",i,cont);
	}
	
	return 0;
}

File: metrica.py
casos = input()
for caso in range(int(casos)):
    vMensal = input()
    ids = input().split()
    resposta = []
    resposta.append(ids[0])
    cont = 1
    for conte in ids:
        existe = False
        for  resp in resposta:
            if conte == resp:
                existe = True        
        if existe == False:
            resposta.append(conte)
            cont = cont + 1
    print("Caso "+str(caso + 1)+": "+str(cont))
